Source/Modules/lang.cxx

```diff
@@ -1320,16 +1320,21 @@ int Language::staticmemberfunctionHandler(Node *n) {
     Delete(mrename);
     mrename = mangled;
 
-    if (Getattr(n, "sym:overloaded") && code) {
-      Append(cname, Getattr(defaultargs ? defaultargs : n, "sym:overname"));
-    }
+    if (code) {
+      // See Swig_MethodToFunction() for the explanation of this code.
+      if (Getattr(n, "sym:overloaded")) {
+	Append(cname, Getattr(defaultargs ? defaultargs : n, "sym:overname"));
+      } else {
+	Append(cname, "__SWIG");
+      }
```

Comment:
If I follow, the only functional change here is to append `__SWIG` if `(!Getattr(n, "sym:overloaded") && code)` - the hunk below is only refactoring to hoist the `code` check to be around all this.

This seems to have potential to affect other target languages - does `(!Getattr(n, "sym:overloaded") && code)` never hold for existing target languages?

Reply:
Oh this seems to be related to #2095

Reply:
#2095 is now merged to master.